src/backend/resourcemanager/resourcemanager_RMSEG.c

```diff
@@ -258,24 +258,8 @@ bool CheckTmpDirAvailable(char *path)
 	{
 		elog(LOG, "Can't open file:%s when check temporary directory", fname);
```

Comment:
elog(LOG, "Cannot open file %s while checking availability of temporary directory: %s",
                  fname,
                  strerror(errno));